src/main/kotlin/controllers/worldRenderer/Renderer.kt

```diff
@@ -995,4 +1028,17 @@ class Renderer @Inject constructor(
         }
         glDeleteBuffers(gl, pboIds)
     }
+
+    private fun checkFpsCap() {
+        val fpsCap = configuration.getProp(SettingsController.FPS_CAP_PROP).toIntOrNull()
+
+        if (fpsCap != null) {
+            setFpsTarget(fpsCap)
+        }
+    }
```

Comment:
Not the responsibility of this class, should go in with the UI controllers

Reply:
Where specifically?

Reply:
WorldRendererController, I think is most appropriate